### PR TITLE
feat: expose prometheus object, resolve #73

### LINKED ISSUE #73: Way to add custom metrics
It would be nice for Kong plugins to be able to add custom metrics to the prometheus endpoint.

At the lowest technical level, this could be done by exposing the `prometheus` object from the "exporter" module.

### PR DESCRIPTION
**Pull request by aldor007**

### CODE REVIEW

**Review by mpacholec — APPROVED** [review]
Looks good 👍

File: kong/plugins/prometheus/exporter.lua [hunk 1]
```diff
@@ -178,10 +178,18 @@ local function collect()
   prometheus:collect()
 end
 
+local function get_prometheus()
+    if not prometheus then
```
Comment (fffonion) [inline kong/plugins/prometheus/exporter.lua]: type: let's do a two-space indention here to keep it consistent

**Review by fffonion — COMMENTED** [review]
Looks good to me other than a small styling issue.

@aldor007 Could you rebase your branch on master so the CI can run again?

**Comment by aldor007** [comment]
@fffonion  
PR updated

**Review by fffonion — APPROVED** [review]

**Pull request merged**